src/ebonite/core/objects/core.py

```diff
@@ -303,13 +304,15 @@ class Model(EboniteObject):
     :param creation_date: date when this model was created
     """
 
-    def __init__(self, name: str, wrapper: ModelWrapper,
+    def __init__(self, name: str, wrapper: Optional[ModelWrapper] = None,
                  artifact: 'ArtifactCollection' = None,
                  requirements: Requirements = None, id: str = None,
                  task_id: str = None,
                  author: str = None, creation_date: datetime.datetime = None):
         super().__init__(id, name, author, creation_date)
-        self.wrapper = wrapper
+
+        self._wrapper = wrapper
+        self._wrapper_json = None
```

Comment:
In my opinion _json is bad, because json is usually a string. _dict or _meta is better.

Reply:
Agree, renaming to `_meta`